modelplace_api/visualization.py

```diff
@@ -67,6 +67,37 @@ def draw_line(
             i += 1
 
 
+def draw_text_label(
+    image: np.ndarray,
+    detection: AgeGenderLabel,
```

Comment:
Shouldn't it be `Union[AgeGenderLabel, EmotionLabel]` there?
Is this function for these two models only?